src/ui/inspector/PlotElementsPool.vue

```diff
@@ -0,0 +1,333 @@
+            >
+                <li
+                    class="js-first-place"
+                    @drop="moveTo($event, 0, Y_AXIS_2)"
+                ></li>
+                <element-item
+                    v-for="(element, index) in yAxis2"
+                    :key="element.identifier.key"
+                    :index="index"
+                    :element-object="element"
+                    :parent-object="parentObject"
+                    :allow-drop="allowDrop"
+                    @dragstart-custom="moveFrom($event, Y_AXIS_2)"
+                    @drop-custom="moveTo($event, index, Y_AXIS_2)"
+                />
+                <li
+                    v-if="yAxis2.length > 0"
+                    class="js-last-place"
+                    @drop="moveTo($event, yAxis2.length, Y_AXIS_2)"
+                ></li>
+            </element-item-group>
+            <element-item-group
+                :parent-object="parentObject"
+                :allow-drop="allowDrop"
+                label="Y Axis 3"
+                @drop-group="moveTo($event, 0, Y_AXIS_3)"
+            >
+                <li
+                    class="js-first-place"
+                    @drop="moveTo($event, 0, Y_AXIS_3)"
+                ></li>
+                <element-item
+                    v-for="(element, index) in yAxis3"
+                    :key="element.identifier.key"
+                    :index="index"
+                    :element-object="element"
+                    :parent-object="parentObject"
+                    :allow-drop="allowDrop"
+                    @dragstart-custom="moveFrom($event, Y_AXIS_3)"
+                    @drop-custom="moveTo($event, index, Y_AXIS_3)"
+                />
+                <li
+                    v-if="yAxis3.length > 0"
+                    class="js-last-place"
+                    @drop="moveTo($event, yAxis3.length, Y_AXIS_3)"
+                ></li>
+            </element-item-group>
+        </ul>
+        <div
+            v-if="yAxis1.length === 0
+                && yAxis2.length === 0
+                && yAxis3.length === 0"
+        >
+            No contained elements
+        </div>
+    </div>
+</div>
+</template>
+
+<script>
+import _ from 'lodash';
+import Search from '../components/search.vue';
+import ElementItem from './ElementItem.vue';
+import ElementItemGroup from './ElementItemGroup.vue';
+
+const Y_AXIS_1 = 1;
+const Y_AXIS_2 = 2;
+const Y_AXIS_3 = 3;
+
+export default {
+    components: {
+        Search,
+        ElementItemGroup,
+        ElementItem
+    },
+    inject: ['openmct'],
+    data() {
+        return {
+            yAxis1: [],
+            yAxis2: [],
+            yAxis3: [],
+            isEditing: this.openmct.editor.isEditing(),
+            parentObject: undefined,
+            currentSearch: '',
+            selection: [],
+            contextClickTracker: {},
+            allowDrop: false
+        };
+    },
+    created() {
```

Comment:
You shouldn't need `created` anymore with the change suggested for using a yAxes list